scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
Hmmm... I thought about this line for instance: replace(':num', cp.l10n.nextSlide); So if we drop the support for :num there we don't need the index parameter anymore. We can use cp.currentSlideIndex in the go function instead, but we're getting into details now. Up to you.